### DISCUSSION TITLE
cloning Folia by PaperMC error

### DISCUSSION THREAD

**Discussion by HoneyBerries** [community/community#79130] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I tried cloning https://github.com/PaperMC/Folia and it cloned but gave me an error. Here is the error:

Cloning into 'C:\Users\whateverMyUsernameIs\Documents\GitHub\Folia\Folia'...
remote: Enumerating objects: 1671, done.        
remote: Counting objects: 100% (1671/1671), done.        
remote: Compressing objects: 100% (604/604), done.        
remote: Total 1671 (delta 1025), reused 1640 (delta 1024), pack-reused 0        
Receiving objects: 100% (1671/1671), 3.08 MiB | 11.78 MiB/s, done.
Resolving deltas: 100% (1025/1025), done.
      0 [main] sh (21712) C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x13E1408/0x1321408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
      0 [main] sh 1635 dofork: child -1 - forked process 21712 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      0 [main] sh (17464) C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x13E1408/0x1351408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
1088194 [main] sh 1635 dofork: child -1 - forked process 17464 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      0 [main] sh (13476) C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x13E1408/0x12E1408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
3180636 [main] sh 1635 dofork: child -1 - forked process 13476 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      0 [main] sh (23772) C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x13E1408/0x13D1408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
7275401 [main] sh 1635 dofork: child -1 - forked process 23772 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: retry: Resource temporarily unavailable
      1 [main] sh (15968) C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\usr\bin\sh.exe: *** fatal error - cygheap base mismatch detected - 0x13E1408/0x13F1408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
15361943 [main] sh 1635 dofork: child -1 - forked process 15968 died unexpectedly, retry 0, exit code 0xC0000142, errno 11
C:\Users\whateverMyUsernameIs\AppData\Local\GitHubDesktop\app-3.3.6\resources\app\git\mingw64\libexec\git-core\git-submodule: fork: Resource temporarily unavailable

Please help

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by HoneyBerries** [reply]
I actually did what the video here https://youtu.be/oTh9DjFgmTM and that error occured when trying to clone the resporitory
